samples/bluetooth/broadcast_audio_assistant/README.rst

```diff
@@ -0,0 +1,51 @@
+.. zephyr:code-sample:: bluetooth_broadcast_audio_assistant
+   :name: Bluetooth: Broadcast Audio Assistant
+   :relevant-api: bt_bap
+
+   Bluetooth: Broadcast Audio Assistant
+
+Overview
+********
+
+Application demonstrating the LE Audio broadcast assistant functionality.
+
+The sample will automatically try to connect to a device in the BAP Scan Delegator
+role (advertising support for the Broadcast Audio Scan Service (BASS)).
+It will then search for a broadcast source and (if found) add the broadcast ID to
+the BAP Scan Delegator.
```

Comment:
Just a thought but this might sound a bit complex for someone incredibly familiar with or interested in the underlying concepts and yet willing to try this "audio broadcast" thing. Since it looks like you've been using with sample in a very practical way with e.g. earbuds, does it make sense to add another paragraph that describes how a user may *practically* give this a try? "A typical way of running this sample would be to ... ... blah blah earbuds ..." :)
Does that make sense?

Reply:
I think it makes sense, thanks